backend/src/components/sdc.js

```diff
@@ -488,6 +488,30 @@ async function getSdcSchoolCollectionStudent(sdcSchoolCollectionStudentID, res,
   return await getData(token, `${config.get('sdc:schoolCollectionStudentURL')}/${sdcSchoolCollectionStudentID}`, correlationID);
 }
 
+async function getSdcSchoolCollections(req, res) {
+  try {
+    const token = getAccessToken(req);
+
+    if (res.locals.requestedSdcDistrictCollection) {
```

Comment:
Please throw a 400 if this attribute is not present and remove the `else` down below (follows our current patterns).